### ISSUE TITLE
multicolor field RESET to default not working totally (color in the field are not resetting)

### ISSUE THREAD

**Issue by thatvideowebguy** [themeum/kirki#916]
### Issue description:

When you click to reset button , the magic append , colors are reseted to default
but the colors within the multicolor field aren't reseted. 
and you can see [object Object] in each color field. 
### Version used:

 develop branch from github
### Using theme_mods or options?

theme_mods
### Code to reproduce the issue (config + field(s))

```
Kirki::add_field( '_s_theme', array(
    'type'        => 'multicolor',
    'settings'    => 'link_color',
    'label'       => esc_attr__( 'Couleurs des liens', 'my_textdomain' ),
    'section'     => 'color',
    'priority'    => 10,
    'choices'     => array(
        'link'    => esc_attr__( 'Couleur', 'my_textdomain' ),
        'hover'   => esc_attr__( 'Survol', 'my_textdomain' ),
        'active'  => esc_attr__( 'Actif', 'my_textdomain' ),
    ),
    'default'     => array(
        'link'    => '#0088cc',
        'hover'   => '#00aaff',
        'active'  => '#00ffff',
    ),
) );

<?php
$defaults = array(
    'link'    => '#0088cc',
    'hover'   => '#00aaff',
    'active'  => '#00ffff',
);
$colors = get_theme_mod( 'link_color', $defaults );
echo '<style>';
echo '#content a { color: ' . $colors['link'] . '!important; }';
echo '#content a:hover { color: ' . $colors['hover'] . '!important; }';
echo '#content a:active { color: ' . $colors['active'] . '!important; }';
echo '</style>';
?>
```

**Comment by aristath** [comment]
This is still a work in progress.
Marking for the 2.4 version 👍

**Comment by willi-dev** [comment]
hello.. 
i have the same issue about multicolor reseted value. 
is it still in progress? will be release for 2.4 version? 

thanks.

**Comment by aristath** [comment]
Fixed in 3.0
